libs/components/storybook/src/lib/preview-wrapper/preview-wrapper-decorators.ts

```diff
@@ -18,7 +19,7 @@ export const previewWrapperDecorators: DecoratorFunction<
   unknown[]
 >[] = [
   moduleMetadata({
-    imports: [CommonModule, PreviewWrapperModule],
+    imports: [CommonModule, PreviewWrapperModule, RouterTestingModule],
```

Comment:
Is there something different in Angular 17 that needed this module imported?

Reply:
Doing this allowed me to drop the dependency on `storybook-addon-angular-router`, which didn't work with the updated Storybook.

Reply:
Cool.